libs/ui/packages/native/src/components/Icon/IconBox/index.tsx

```diff
@@ -5,6 +5,8 @@ import Flex, { FlexBoxProps } from "../../Layout/Flex";
 
 type Props = {
   Icon: IconOrElementType;
+  variant?: "outlined" | "plain";
```

Comment:
The IconBox component is deprecated :grimacing: 
You have to modify BoxedIcon instead ... which already has a round variant
Sorry, there is a lack of documentation on this ... we should have deleted IconBox already

Reply:
Done